packages/spec-xunit-reporter/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "spec-xunit-reporter",
+	"version": "0.0.1",
+	"description": "Mocha reporter that combines the default spec output with an xunit.xml file in a dynamically allocated directory",
+	"main": "index.js",
+	"author": "Automattic Inc.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/spec-xunit-reporter"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"private": true,
+	"peerDependencies": {
+		"mocha": "^5.2.0"
```

Comment:
Originally, this package was depending on

```
		"config": "^1.28.0",	
		"mocha": "^5.2.0",	
		"fs-extra": "^0.22.1"
```

`config` and `fs-extra` are not used so I dropped them.`mocha` is better suited as a `peerDependency`, as we are building a plugin for `mocha`.